spec/system/candidate_interface/carry_over/candidate_carries_over_different_application_states_spec.rb

```diff
@@ -8,6 +8,26 @@
     @application_form = create(:application_form, :completed, :pre_continuous_applications, candidate: @candidate)
   end
 
+  # To avoid infinite redirects for "QA" data or any other data states that
+  # might cause odd behaviour on carry over
+  scenario 'Candidate sees complete page when submitted at has value but courses choices does not have submission' do
+    given_i_have_an_empty_submitted_application_from_last_cycle
+    and_i_am_signed_in_as_a_candidate
+    when_i_sign_in_again
+    and_i_visit_the_application_dashboard
+    then_i_should_be_on_the_pages_that_is_possible_to_carry_over_an_application
+  end
+
+  scenario 'Candidate carries over empty application to new cycle through the carry over insterstitial' do
```

Comment:
`insterstitial` typo, and the spec below.

Reply:
Good spot. Fixed